app/views/devise/registrations/new.html.slim

```diff
@@ -15,30 +15,43 @@ body
                   span.input-group-text
                     i.fas.fa-user
                 .field
-                  = f.text_field :first_name, autofocus: true, placeholder: 'first name', autocomplete: 'first_name'
+                  = f.text_field :first_name,
+                             autofocus: true,
+                             placeholder: 'first name',
+                             autocomplete: 'first_name'
```

Comment:
strange alignment

Reply:
= f.text_field :first_name,
                     autofocus: true,
                     placeholder: 'first name',
                     autocomplete: 'first_name'

this alignment look good?

Reply:
`= f.text_field :first_name,
                      autofocus: true,
                      placeholder: 'first name',
                      autocomplete: 'first_name`

Reply:
done